doc/api-extensions.md

```diff
@@ -65,6 +65,9 @@ Add support for the HTTP PATCH method.
 
 PATCH allows for partial update of an object in place of PUT.
 
+## usb\_devices
+Add support for USB hotplug.
+
```

Comment:
Can we get that in a separate commit so I can put it in the 2.4.2 bugfix release?